src/Database/Database.php

```diff
@@ -160,6 +165,11 @@ public function connect()
 				if ($charset) {
 					$this->connection->set_charset($charset);
 				}
+                                
+                                if ($socket) {
+                                        $this->connection->set_socket($socket);
+                                }
+
```

Comment:
```suggestion

				if ($socket) {
					$this->connection->set_socket($socket);
				}

```

Code standard - wrong indent